core/services/helper/main.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import http
+import urllib
+from typing import List
+from urllib.request import urlopen
+
+import bottle
+import psutil
+
+PORT = 8080
```

Comment:
shouldn't this be 80?

Reply:
Sorry, the README was wrong, the correct is 8080, I thought that 80 should be reserved for our main page.